### PR TITLE
Solving #38

### PR DESCRIPTION
**Pull request by gabrielecker**
Fixing #38 
Added an english version of README, open to advices in case you want to change something.

### CODE REVIEW

File: README.en-US.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+![Tela2](http://i.imgur.com/XDa192l.png)
+
+## What is the currently technology stack?
+
+Front-end:
+* [React](https://facebook.github.io/react/)
+* [Redux](http://redux.js.org/)
+* [Semantic UI React](http://react.semantic-ui.com)
+* [Axios](https://github.com/mzabriskie/axios)
+* [Immutable](https://facebook.github.io/immutable-js/)
```
Comment (mblodorn) [inline README.en-US.md]: We need to remove Immutable from the doc in the brazilian and english version, it's not being used anymore. Not sure if I should open a specific issue for that.

File: README.en-US.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+### Back-end:
+
+Still not available.
+
+## Development achievements:
+
+- [x] Consolidate front-end structure.
+- [x] Begin initial (MVP) components development (front-end).
+- [ ] Begin initial server development (back-end).
```
Comment (mblodorn) [inline README.en-US.md]: We can mark that as done, i've started the back-end at https://github.com/animapp-dev/animapp-server

**Comment by mblodorn** [comment]
Thanks for you contribution! I pointed out few things that we may change. Tell me what do you think. :)

File: README.en-US.md (hunk 1 above)
Comment (gabrielecker) [inline README.en-US.md]: Maybe open a general issue relating all those small fixes :thinking:

File: README.en-US.md (hunk 2 above)
Comment (gabrielecker) [inline README.en-US.md]: The same pointed out on the previous comment applies to this I think

**Comment by gabrielecker** [comment]
You're welcome :smile: 
I had a similar project idea but yours seems waaay more mature. I'll be studying this one to contribute with more effectiveness in the future :+1:

File: README.en-US.md (hunk 1 above)
Comment (gabrielecker) [inline README.en-US.md]: Or we just fix everything right there and open new ones in the future if needed

**Comment by mblodorn** [comment]
That would be great! I want to update this project and put it in production ASAP :)

File: README.en-US.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+Animapp is an online and open-source platform developed in favor of the animal cause. Through this platform you may adopt a pet, register yourself as temporary shelter owner or donate to animals in need. If you found a helpless pet that needs help, you may register him as well.
+
+Like our Facebook page! [http://www.facebook.com/animapp.dev](http://www.facebook.com/animapp.dev)
+
+![Mascote](http://i.imgur.com/HYRreHI.png)
+![Tela1](http://i.imgur.com/GskS6NH.png)
+![Tela2](http://i.imgur.com/XDa192l.png)
+
+## What is the currently technology stack?
```
Comment (mblodorn) [inline README.en-US.md]: "What is the current technology stack?" sounds better in my opinion, forgot to comment this one, my bad!

Comment (mblodorn) [inline README.en-US.md]: I think this should be "What is the current technology stack?"

**Pull request merged**